### ISSUE TITLE
Import vs Import (parse only) is dropping some resets

### ISSUE THREAD

**Issue by TokugawaTFC** [Gwyrdain/apprentices-workshop#127]
I've sent you the area file so you can debug this. I imported the zone Te'nayu'ka into the editor. Cordir noticed two E resets were missing. The resets missing are below:

M 0 19872 1 19857  * a drow elf passenger -> The Passengers' Berth
E 0 19869 100 15  * leather vambraces with spiderweb repousse
E 0 19868 100 5  * an over-robe of emerald silkweave

Below are two screenshots: The parse (import only) shows the resets. The import review shows them missing.

![import-only](https://cloud.githubusercontent.com/assets/11012495/15491079/fc1957ae-2121-11e6-9fc7-0f50c77634a9.png)
![import-parse-only](https://cloud.githubusercontent.com/assets/11012495/15491080/fc199a52-2121-11e6-984a-0b3a2c18f393.png)

**Comment by Gwyrdain** [comment]
It looks like those resets were "correctly" rejected.

Vambraces: Object of type "ARMS" being reset to the "RIGHT WRIST" position (15) -- should be ARMS (10)
Over-robe:  Object of type "ABOUT" being reset to the "BODY" position (5) -- should be ABOUT BODY (12)

Further thought is needed regard how best to deal with a situation like this.
1.  Reject with some form of error reporting?  A reporting framework is not in place.
2.  Auto-fix and report what was done?  Framework again.

Out of curiosity -- did these equips work in game?

**Comment by TokugawaTFC** [comment]
Yep, they worked. That's too bad about the import not displaying what it rejected. I don't think we can use the import if we do not know about items being rejected.

A drow elf passenger is using:
<worn on arms>      (Moderate magic) leather vambraces with spiderweb repousse
<worn about body>   (Moderate magic) an over-robe of emerald silkweave

**Comment by Gwyrdain** [comment]
I understand and agree.  Not sure how to deal with it yet.
